opal/mca/btl/tcp/btl_tcp_component.c

```diff
@@ -732,8 +732,8 @@ static int mca_btl_tcp_component_create_instances(void)
         char* if_name = *argv;
         int if_index = opal_ifnametokindex(if_name);
         if(if_index < 0) {
-            opal_show_help("help-mpi-btl-tcp.txt", "invalid if_inexclude",
-                           true, if_name, opal_process_info.nodename,
+            opal_show_help("help-mpi-btl-tcp.txt", "invalid interface",
```

Comment:
I liked the old message better.  I just think you wanted to use "include" instead of if_name as the first positional argument.  The problem with this error message is that it won't be clear to the user where they specified the invalid interface name.